tests/functional/csp-cdn.ts

```diff
@@ -0,0 +1,13 @@
+require.config({
+	packages: [
+		{name: 'amdApp', location: './amdApp'},
+		{name: 'dojo', location: '//ajax.googleapis.com/ajax/libs/dojo/1.10.4/dojo'}
```

Comment:
fwiw, there is https://ajax.googleapis.com/ajax/libs/dojo/1.11.2/dojo/dojo.js at least on CDN if you want to test something more recent. :)